### PR TITLE
Cloud changelog March 24th

### PR DESCRIPTION
**Pull request by amyblais**
See below outstanding question for PM https://github.com/mattermost/docs/pull/4390#issuecomment-801981996

### CODE REVIEW

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 1]
```diff
@@ -2,6 +2,35 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-03-24
+
+### Improvements
+
+#### User Interface (UI)
+ - Added a UI field for restricted domains under **Site Configuration > Posts** in the System Console. Expanded the logic that determines whether a post has a preview or not.
+ - Added support for auto-RTL detection in browsers.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:11]:
```suggestion
 - Added support for automatic right-to-left (RTL) detection in browsers.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 2]
```diff
@@ -2,6 +2,35 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-03-24
+
+### Improvements
+
+#### User Interface (UI)
+ - Added a UI field for restricted domains under **Site Configuration > Posts** in the System Console. Expanded the logic that determines whether a post has a preview or not.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:10]: @amyblais Can you tell me which feature PR this is driven by, please? (It's not yet on my radar.)

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 3]
```diff
@@ -2,6 +2,35 @@
...
+ - Added support for auto-RTL detection in browsers.
+ - Updated the font size for the **Add People** channel modal.
+ - Online status is now shown in the channel switcher.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:13]: @amyblais May I have the feature PR driving this line too, please? Thanks!

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 4]
```diff
@@ -2,6 +2,35 @@
...
+ - Improved the design and layout of email notifications.
+
+#### Administration
+ - Profiling the Mattermost server with pprof is now available for Team Edition.
+ - Added attributes to split.io feature flags.
+
+### Bug Fixes
+ - Fixed bugs related to replication lag for Enterprise Edition instances configured to use read replicas.
+ - Fixed an issue where Compliance report field headers were not correctly aligned.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:22]:
```suggestion
 - Fixed an issue where Compliance Report field headers were not correctly aligned.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 5]
```diff
@@ -2,6 +2,35 @@
...
+ - Fixed an issue where the ``/join`` command was case-sensitive.
+ - Fixed an issue where one-character sidebar category names were not displayed.
+ - Fixed an issue with a theme discrepancy on close buttons on some modals in the System Console (when using custom team theme).
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:25]:
```suggestion
 - Fixed an issue with a theme discrepancy on close buttons on some modals in the System Console (when using a custom team theme).
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 6]
```diff
@@ -2,6 +2,35 @@
...
+ - Fixed an issue where the text draft area in the right-hand side was jumpy with long text input when selected.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:26]:
```suggestion
 - Fixed an issue where long text input in the right-hand pane was jumpy when selected.
```

**Review by cwarnermm — COMMENTED** [review]
Suggestions provided inline.

File: source/cloud/cloud-administration/cloud-changelog.md (hunk 3 above)
Comment (amyblais) [reply source/cloud/cloud-administration/cloud-changelog.md:13]: @cwarnermm https://github.com/mattermost/mattermost-webapp/pull/6609

File: source/cloud/cloud-administration/cloud-changelog.md (hunk 2 above)
Comment (amyblais) [reply source/cloud/cloud-administration/cloud-changelog.md:10]: @cwarnermm https://github.com/mattermost/mattermost-server/pull/16869

**Comment by amyblais** [comment]
@wiersgallak For the 2 new features, are these Cloud-specific features or do they also exist in on-prem, and do we want to mention these in the public changelog or just in the internal changelog?
 - A re-designed admin welcome email is now sent after a Cloud installation is complete.
 - Added plugins and incident collaboration in the ‘Tips & Next Steps’ section for admins.

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 7]
```diff
@@ -2,6 +2,37 @@
...
 
+## Release 2021-03-24
+
+### Improvements
+
+#### User Interface (UI)
+ - Added support for automatic right-to-left (RTL) detection in browsers.
+ - Updated the font size for the **Add People** channel modal.
+ - Online status is now shown in the channel switcher.
+ - Improved the design and layout of email notifications.
```
Comment (esethna) [inline source/cloud/cloud-administration/cloud-changelog.md:13]: let's be specific about which types of email notifications were updated, not all types were. It should be in the PR

**Review by esethna — APPROVED** [review]

File: source/cloud/cloud-administration/cloud-changelog.md (hunk 7 above)
Comment (amyblais) [reply source/cloud/cloud-administration/cloud-changelog.md]: Updated.

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 8]
```diff
@@ -2,6 +2,37 @@
...
 
+## Release 2021-03-24
+
+### Improvements
+
+#### User Interface (UI)
+ - Added support for automatic right-to-left (RTL) detection in browsers.
+ - Updated the font size for the **Add People** channel modal.
+ - Online status is now shown in the channel switcher.
+ - Improved the design and layout of email notifications for password resets, member invites, and member welcome and verifications.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:13]:
```suggestion
 - Improved the design and layout of email notifications for password resets, member invites, member welcome, and verifications.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 9]
```diff
@@ -2,6 +2,37 @@
...
+
+#### Administration
+ - Added mmctl commands to create, list, download and delete export files.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:16]:
```suggestion
 - Added ``mmctl`` commands to create, list, download, and delete export files.
```

**Review by cwarnermm — APPROVED** [review]
Non-blocking suggestions provided inline.

**Pull request merged**